tests/utils.py

```diff
@@ -26,8 +26,8 @@ def is_item_available(item_pid):
     return True
 
 
-def can_be_requested(item_pid=None, document_pid=None):
-    """Return if record can be requested."""
+def can_be_requested(loan):
```

Comment:
I think this is not used somewhere.

Reply:
It's used for testing. [https://github.com/inveniosoftware/invenio-circulation/blob/master/tests/conftest.py#L32](url)